geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DescribeMappingCommand.java

```diff
@@ -102,38 +102,7 @@ public ResultModel describeMapping(@CliOption(key = DESCRIBE_MAPPING__REGION_NAM
 
   private CacheConfig getCacheConfig(ConfigurationPersistenceService configService, String group)
```

Comment:
Do we still need this private method `getCacheConfig`? Since we can directly call `MappingCommandUtils.getCacheConfig(configService, group)`.

Reply:
✅